server/routes/journeys/base/updateReferralContributoryFactorController.ts

```diff
@@ -0,0 +1,68 @@
+import { NextFunction, Request, Response } from 'express'
+import { SanitisedError } from '../../../sanitisedError'
+import {
+  FLASH_KEY__CSIP_SUCCESS_MESSAGE,
+  FLASH_KEY__FORM_RESPONSES,
+  FLASH_KEY__VALIDATION_ERRORS,
+} from '../../../utils/constants'
+import { PatchReferralController } from './patchReferralController'
+import { ContributoryFactor } from '../../../@types/express'
+import { getNonUndefinedProp } from '../../../utils/utils'
+
+export const MESSAGE_REFERRAL_DETAILS_UPDATED = 'You’ve updated the referral details.'
+export const MESSAGE_REACTIVE_DETAILS_UPDATED = 'You’ve updated the incident details.'
+export const MESSAGE_PROACTIVE_DETAILS_UPDATED = 'You’ve updated the behaviour details.'
+export const MESSAGE_REACTIVE_INVOLVEMENT_UPDATED = 'You’ve updated the incident involvement information.'
+export const MESSAGE_PROACTIVE_INVOLVEMENT_UPDATED = 'You’ve updated the behaviour involvement information.'
+export const MESSAGE_REACTIVE_DESCRIPTION_UPDATED = 'You’ve updated the incident description.'
+export const MESSAGE_PROACTIVE_DESCRIPTION_UPDATED = 'You’ve updated the behaviour description.'
+export const MESSAGE_CONTRIBUTORY_FACTOR_UPDATED = 'You’ve updated the information on contributory factors.'
+export const MESSAGE_REFERRAL_ADDITIONAL_INFO_UPDATED = 'You’ve updated the additional information.'
```

Comment:
just use the constants from `PatchReferralController`? don't need to repeat them here.

Reply:
lazy copy/paste. Good spot